core/store/migrations/migration1574659987/migrate.go

```diff
@@ -2,15 +2,16 @@ package migration1574659987
 
 import (
 	"github.com/jinzhu/gorm"
-	"github.com/pkg/errors"
-
-	"github.com/smartcontractkit/chainlink/core/store/models/vrfkey"
 )
 
-// Migrate adds VRF proving-key table, and related subtables.
+// Migrate adds VRF proving-key table
 func Migrate(db *gorm.DB) error {
-	if err := db.AutoMigrate(&vrfkey.EncryptedSecretKey{}).Error; err != nil {
-		return errors.Wrap(err, "failed to create VRF proving-key table")
-	}
-	return nil
+	return db.Exec(`
+		CREATE TABLE encrypted_secret_keys (
+			public_key character varying(68) PRIMARY KEY,
+			vrf_key text NOT NULL,
+			created_at timestamp with time zone NOT NULL,
+			updated_at timestamp with time zone NOT NULL
+		);
+	`).Error
```

Comment:
Why did you change this older migration?

Reply:
Because it was using gorm's horrible automigrate feature which changes when you update the struct. I changed it to be what it WOULD have always done.